openid-4-verifiable-presentations-1_0.md

```diff
@@ -1704,6 +1704,17 @@ Note: If the Verifier's Response URI did not return a `redirect_uri` in step (6)
 
 (10) The Verifier checks whether the `nonce` received in the Credential(s) in the VP Token in step (9) corresponds to the `nonce` value from the session. The Verifier then consumes the VP Token and invalidates the `transaction-id`, `request-id` and `nonce` in the session.
 
+## Pre-Final Specifications
+
+Implementers should be aware that this specification uses several specifications that are not yet final specifications. Those specifications are:
+
+- OpenID Federation 1.0 draft -42 [@!OpenID.Federation]
+- SIOPv2 draft -13 [@!SIOPv2]
+- SD-JWT-based Verifiable Credentials (SD-JWT VC) draft -08 [I-D.ietf-oauth-sd-jwt-vc]
+- W3C Digital Credentials API - [@!W3C.Digital_Credentials_API]
+
+While breaking changes to these specifications are not expected, should they occur, OpenID4VP implementations should continue to use the specifically referenced draft versions above in preference to the final versions, unless updated by a profile or new version of this specification.
```

Comment:
It might be worth clarifying whether this applies to non-draft versions too? e.g. we reference ISO/IEC 18013-5:2021 - is that the only one you can use? (Whereas for the BCP195 reference I'm relatively sure we definitely would expect people to follow the latest version.)

Reply:
can we do that in a different PR? the purpose of this PR is to clarify draft specs.

Reply:
I guess we could.

Reply:
WG discussion: 
make clear 
- a specific version to be used with openid4vp final.
- allow using updated versions of already published specifications if there is no breaking changes
- do an errata when an updated version of these specs are published
 
```suggestion
While breaking changes to these specifications are not expected, should they occur, OpenID4VP implementations should continue to use the specifically referenced versions above in preference to the final versions, unless updated by a profile or new version of this specification.
```